index.html

```diff
@@ -0,0 +1,111 @@
+        <header> Introduction brief</header>
+        <article>
+            <p >
+JavaScript is a cross-platform, object-oriented scripting language
+            </p>
+            <p >
+                JavaScript is a cross-platform, object-oriented scripting language
+                            </p>
+                            
+        </article>
+    </section>
+    <section class="main-section" id="What_you_should_already_know">
+        <header>What you should already know</header>
+        <article>
+            <p>
+       This guide assumes you have the following basic background  
+            </p>
+            <p>
+                This guide assumes you have the following basic background  
+                     </p>
+        </article>
+    </section>
+    <section class="main-section" id="JavaScript_and_Java">
+        <header>JavaScript and Java</header>
+        <article>
+            <p>
+The JavaScript language resembles Java but does not have Java's static typing and strong type checking.
+            </p>
+            <code> cout<<"hhello;</code>
+            <ul>
+                <li>syntax</li>
+                <li>naming conventions </li>
+                <li>basic control-flow constructs </li>
+            </ul>
+            <p>
+                This guide assumes you have the following basic background  
+                     </p>
+
+        </article>
+    </section>
+    <section class="main-section" id="Hello_world">
+        <header>Hello world</header>
+        <article>
+            <p>
+ To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:  
+            </p>
+            <p>
+                To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code:  
+                           </p>
+            <code> system.out.ptint()</code>
+            <code> cin<<"hhello;</code>
+            <code>y=34.4</code>
+        </article>
+    </section>
+    <section class="main-section" id="Variables_two">
+        <header>Variables two</header>
+        <article>
+            <p>
+                You use variables as symbolic names for values in your application.  
+            </p>
+            <ul>
+                <li>global</li>
+                <li>local </li>
+                <li>scope </li>
+            </ul>
+        </article>
+    </section>
+    <section class="main-section" id="Declaring_variables">
+        <header>Declaring variables</header>
+        <article>
+            <p>
+                You can declare a variable in three ways:
+            </p>
+            <code> let x;</code>
+            <code> var y=0;</code>
+            <code>int y=0;</code>
+            <p>
+                this is C++ declaration
+            </p>
+            </article>
+        
+    </section>
+  </main>
+<!-- <nav class="navbar">
+    <header>JAva docume </header>
+  
+-->
```

Comment:
- [ ] Good job so far ' 👏 ', but kindly remove any commented code that isn't needed from your project. Your codebase becomes unnecessarily cluttered when unneeded code is commented. In particular, it makes the code more difficult to read and comprehend for future engineers who could work on the project. It can cause confusion and make it difficult to distinguish the active section of your code. 🙂

Reply:
- [ ] Great work adding these comments ✔️ . Its polite and clearly explains the reason for change